signing_clients/tests/__init__.py

```diff
@@ -129,3 +134,11 @@ def test_08_unicode(self):
                                  'signing_clients/tests/test-jar-unicode-signed.jar',
                                  omit_signature_sections=False)
         self.assertEqual(str(extracted.manifest), UNICODE_MANIFEST)
+
+    def test_09_serial_number_extraction(self):
```

Comment:
Can you add a comment about how this der file was generated in case someone needs to do it again?